package/wpe/wpeframework-amazon/wpeframework-amazon.mk

```diff
@@ -20,6 +20,10 @@ WPEFRAMEWORK_AMAZON_CONF_OPTS += \
     -DPLUGIN_AMAZON_PRIME_MODEL_NAME=${BR2_PACKAGE_WPEFRAMEWORK_AMAZON_MODEL} \
     -DPLUGIN_AMAZON_PRIME_CHIPSET_NAME=${BR2_PACKAGE_WPEFRAMEWORK_AMAZON_CHIPSET}
 
+ifeq ($(BR2_CMAKE_HOST_DEPENDENCY),)
```

Comment:
What is the issue? Can't we fix this in CMake itself?

Reply:
In that case, $(HOST_DIR)/share/cmake/Modules is not available for the package hence it will not get lib/cmake from this path